### PR TITLE
feat(atom/label): new label component

### PR DESCRIPTION
**Pull request by nguasch**
We have created a new atom label component.

### CODE REVIEW

File: components/atom/label/src/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+AtomLabel.displayName = 'AtomLabel'
+
+AtomLabel.defaultProps = {
+  optional: '(optional)'
+}
+
+AtomLabel.propTypes = {
+  name: PropTypes.string.isRequired,
+  children: PropTypes.node,
```
Comment (jcger) [inline components/atom/label/src/index.js]: You can remove this at it looks like its not being used

File: components/atom/label/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# AtomLabel
+
+> Description
```
Comment (jcger) [inline components/atom/label/README.md]: Please add a description

File: components/atom/label/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+### Basic usage
+```js
+import AtomLabel from '@s-ui/react-atom-label'
+
+return (<AtomLabel />)
+```
+
+
+> **Find full description and more examples in the [demo page](#).**
```
Comment (jcger) [inline components/atom/label/README.md]: please the future link to the demo page `https://sui-components.now.sh/workbench/atom/label/demo`

File: components/atom/label/src/index.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  text: PropTypes.string.isRequired,
+  optional: PropTypes.string,
```
Comment (jcger) [inline components/atom/label/src/index.js]: what about renaming as `subText` or something like that?

File: components/atom/label/src/index.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+)
+
+AtomLabel.displayName = 'AtomLabel'
+
+AtomLabel.defaultProps = {
+  optional: '(optional)'
+}
+
+AtomLabel.propTypes = {
+  name: PropTypes.string.isRequired,
```
Comment (jcger) [inline components/atom/label/src/index.js]: please add the description to the components like done in other components https://github.com/SUI-Components/sui-components/blob/master/components/atom/button/src/index.js#L88

File: components/atom/label/package.json [hunk 6]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@s-ui/react-atom-label",
```
Comment (JoanClaret) [inline components/atom/label/package.json:2]: i was wondering if package name should be `@s-ui/sui-atom-label`

looks like most of the packages use `sui` prefix, even if there are some with `react`:
https://github.com/SUI-Components/sui-components/blob/master/components/atom/spinner/package.json#L2
https://github.com/SUI-Components/sui-components/blob/master/components/atom/badge/package.json#L2

Comment (jcger) [reply components/atom/label/package.json:2]: Enablers decided to rename the components to `@s-ui/react-[category]-[name]` but it isn't mandatory until we do the proper changes in the studio to automatically set the right name

File: components/atom/label/src/index.js (hunk 4 above)
Comment (nguasch) [reply components/atom/label/src/index.js]: We renamed as a `optionalText`

**Review by jcger — APPROVED** [review]

**Review by kikoruiz — APPROVED** [review]

**Review by danivm — APPROVED** [review]

**Review by carlesnunez — APPROVED** [review]

**Pull request merged**